.gitignore

```diff
@@ -30,6 +30,7 @@ tmp/
 build/
 out/
 web3j/
+!**/executionclient/web3j/
```

Comment:
We can probably remove this ignore entirely. I think it used to be there because we generated the deposit contract APIs via web3j but we don't do that anymore.

Reply:
Great, will remove it then. Would be cleaner.